roles/ansible-test/tasks/main.yaml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Include branch specific variables.
+  include_vars: "{{ item }}"
+  with_first_found:
+    - '{{ ansible_test_git_branch }}.yaml'
+    - default.yaml
+
+- name: Prepare ansible-test parameters
+  import_tasks: init_test_options.yaml
+
+- name: Check if galaxy.yml exists
+  stat:
+    path: "{{ ansible_test_location }}/galaxy.yml"
+  register: galaxy_stat_result
+
+- name: Single ansible mode
+  import_tasks: init_collection.yaml
+  when: not galaxy_stat_result.stat.exists
```

Comment:
lets keep ansible_test_collections for this check, not the galaxy file. As there is going to be settings we need to toggle in ansible-test options and don't want to always check if galaxy.yaml exists.

Reply:
Ok, I don't mind putting that back. It tried to reduce the number of role parameter. I'm under the impression we will only test collection soon. @pabelanger, Am I wrong?

Reply:
Yes, soon. We still need to test stable-2.9 and stable-2.8, but we can flip the logic once ansible/ansible devel has been unfrozen

Reply:
Oh. I forgot about the old stable branches.